scss/mixins/_accessibility.scss

```diff
@@ -0,0 +1,14 @@
+//-----------------------------------
+//
+// Accessibility mixins
+// ====================
+//
+//-----------------------------------
+
+// Assert that the parent selector matches one of the selectors passed into $selectors
+@mixin assert-selector($selectors...) {
+  &:not(#{$selectors}) {
+    /*! You must ensure the element matches one of these selectors: #{$selectors} */
+    outline: 2px solid red !important;
```

Comment:
@andrewliebchen oh, although, this will spit out the warning every time the mixin is called. The warning is only useful if they violate it.